py-polars/polars/functions/range.py

```diff
@@ -556,15 +554,146 @@ def date_range(
         s = F.select(result).to_series()
         if s.len() == 1:
             s = s.explode().set_sorted()
+        else:
+            _warn_for_deprecation_date_range()
```

Comment:
there's no test case that reaches this (nor the `time_range` counterpart), any chance to add a little one which does?

Reply:
I'll add that when I get home tomorrow!